store/sqlstore/utils.go

```diff
@@ -26,3 +30,11 @@ func MapStringsToQueryParams(list []string, paramPrefix string) (string, map[str
 
 	return fmt.Sprintf("(%v)", keys.String()), params
 }
+
+// finalizeTransaction ensures a transaction is closed after use, rolling back if not already committed.
+func finalizeTransaction(transaction *gorp.Transaction) {
```

Comment:
Opted for the simpler version of `finalizeTransaction` that wouldn't require much rewriting of error handling in the above usages.